repos/tap/components/results/resultsTabs.js

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback, useEffect, useState, useMemo } from 'react'
+import { View, Button } from '@keg-hub/keg-components'
+import { isFunc, deepMerge, noPropArr, noOpObj } from '@keg-hub/jsutils'
+import { RunTestsButton, Tabbar } from 'SVComponents'
+
+
+/**
+ * TestActions Component - Displays results screen actions for updating test files
+ * @param {Object} props
+ */
+const TestActions = props => {
+  const {
+    actionStyles=noOpObj,
+    onRun,
+  } = props
+
+  return (
+    <View
+      style={actionStyles?.main}
+      className={`results-tab-actions`}
+    >
+      <View
+        style={actionStyles?.run}
+        className={`results-tab-actions-save`}
+      >
+        <RunTestsButton
+          onRun={onRun}
+          runAllTests={false}
```

Comment:
probably would be better to make this configurable through a prop, defaulting to `false`

Reply:
Yah, I plan too. At somepoint we will need to be able to run all features, not just one at a time. So that's what this is for. I just haven't made it any further then this